src/main/kotlin/com/linkedplanet/ktor/Saml.kt

```diff
@@ -59,15 +53,13 @@ fun ApplicationCall.getServletRequest(): HttpServletRequest {
     return servletRequest
 }
 
-@OptIn(EngineAPI::class)
 fun ApplicationCall.getServletResponse(): HttpServletResponse {
     val servletApplicationResponse = getAsyncServletApplicationCall().response
     val responseField = ServletApplicationResponse::class.java.getDeclaredField("servletResponse")
     responseField.isAccessible = true
     return responseField.get(servletApplicationResponse) as HttpServletResponse
 }
 
-@OptIn(EngineAPI::class)
 private fun ApplicationCall.getAsyncServletApplicationCall(): AsyncServletApplicationCall {
     val routingApplicationCall = (request.call as RoutingApplicationCall)
     val callField = RoutingApplicationCall::class.java.getDeclaredField("call")
```

Comment:
Does this still work in the new version of ktor?

Reply:
Alright I've checked it with ktor 2.1.3 and this method still works with the `call` field on the `RoutingApplicationCall` containing a `JettyApplicationCall` (given jetty as engine), which extends the `AsyncServletApplicationCall`